src/components/pages/annual-report-2018/AnnualReportView.js

```diff
@@ -0,0 +1,331 @@
+import React from "react";
+
+import "./AnnualReportView.scss";
+import ShelterTechAnnualReport2018 from "./assets/ShelterTech-Annual-Report-2018.pdf";
+import ImgAskDarcelLogo from "./assets/askdarcel-logo.png";
+import ImgCaseyGraph from "./assets/casey-graph.png";
+import ImgCaseyLogo from "./assets/casey-logo.png";
+import ImgShelterConnectLogo from "./assets/shelterconnect-logo.png";
+
+const AnnualReportView = () => (
+  <div className="annual-report">
+    <div className="content">
+      <div className="content-bg" />
+      <div className="annual-report--overview">
+        <div className="wrapper">
+          <p className="annual-report--year">2018</p>
+          <p className="annual-report--hero">ShelterTech Annual Report</p>
+          <p className="annual-report--text">
+            ShelterTech is an award winning non-profit solving the biggest
+            technology challenges faced by people experiencing homelessness.
+          </p>
+          <p className="annual-report--text">
+            Founded in 2016, ShelterTech is now a 100 member strong
+            volunteer-only organisation with over $100,000 in annual funding,
+            $800,000 in in-kind contributions, and 3 program offerings.
+          </p>
+          <p className="annual-report--text">
+            In 2018, we faced a serious challenge as founders and key leaders
+            moved out of state. Could we build a resilient organisation, able to
+            deliver world-class technology products for homeless communities,
+            all while completely volunteer led and operated? The short answer
+            is: Yes! We tripled in size, outpaced all our goals, and expanded
+            all our programs. We can’t wait for an exciting 2019.
+          </p>
+          <div className="annual-report--growth">
+            <h2>
+              ShelterTech <span className="blue">tripled</span>
+              <br />
+              in size in 2018
+            </h2>
+          </div>
+          <div className="annual-report--goals">
+            <h2>ShelterTech exceeded goals across every program</h2>
+            <ul>
+              <li>
+                <span className="goal-number">7</span>
+                <p>ShelterConnect Wifi Installations</p>
+              </li>
+              <li>
+                <span className="goal-number">7</span>
+                <p>AskDarcel Releases</p>
```

Comment:
@derekfidler, do you have changes that haven't been pushed up to the master branch? I noticed that on the actual live site today, this has been renamed to "SF Service Guide Releases", but the [code on master still shows "AskDarcel Releases"](https://github.com/ShelterTechSF/sheltertech.org/blob/07a64b6ed754cbfb005985dbed3e52987f62b327/src/routes/AnnualReport2018/components/AnnualReportView.js#L32).

Could you please push up any changes that you have locally. It'd be bad if we missed something as we wrap up the code porting changes due to not having the latest changes up on the master branch.